src/main/java/org/zalando/zmon/notifications/NotificationServiceApplication.java

```diff
@@ -55,9 +56,17 @@
     EscalationConfigSource escalationConfigSource;
 
     @Bean
-    TokenInfoService getTokenInfoService(NotificationServiceConfig config, PreSharedKeyStore preSharedKeyStore) {
-        ChainedTokenInfo info = new ChainedTokenInfo(new PreSharedTokenInfoService(preSharedKeyStore),
-                                                     new OAuthTokenInfoService(config.getOauthInfoServiceUrl()));
+    TokenInfoService getTokenInfoService(final NotificationServiceConfig config, final PreSharedKeyStore preSharedKeyStore) {
+        final TokenInfoService info;
+        if(config.isDryRun()) {
+            info = new PreSharedTokenInfoService(new PreSharedKeyStore(
+                    ImmutableMap.of(DRY_RUN_PRE_SHARED_KEY, Long.MAX_VALUE)));
```

Comment:
not sure why you did this, you could just have added one preshared key into application yaml for the test profile i believe.